rotkehlchen/chain/evm/decoding/spark/lend/decoder.py

```diff
@@ -0,0 +1,48 @@
+from collections.abc import Callable, Sequence
+from typing import TYPE_CHECKING
+
+from rotkehlchen.chain.evm.decoding.aave.v3.decoder import Aavev3LikeCommonDecoder
+from rotkehlchen.chain.evm.decoding.spark.constants import (
+    CPT_SPARK,
+    SPARK_COUNTERPARTY_DETAILS,
+)
+from rotkehlchen.chain.evm.decoding.spark.decoder import SparkCommonDecoder
+from rotkehlchen.chain.evm.decoding.types import CounterpartyDetails
+
+if TYPE_CHECKING:
+    from rotkehlchen.chain.evm.decoding.base import BaseDecoderTools
+    from rotkehlchen.chain.evm.decoding.types import CounterpartyDetails
+    from rotkehlchen.chain.evm.node_inquirer import EvmNodeInquirer
+    from rotkehlchen.types import ChecksumEvmAddress
+    from rotkehlchen.user_messages import MessagesAggregator
+
+
+class SparklendCommonDecoder(Aavev3LikeCommonDecoder, SparkCommonDecoder):
+    def __init__(
+            self,
+            evm_inquirer: 'EvmNodeInquirer',
+            base_tools: 'BaseDecoderTools',
+            msg_aggregator: 'MessagesAggregator',
+            pool_addresses: Sequence['ChecksumEvmAddress'],
+            native_gateways: 'tuple[ChecksumEvmAddress, ...]',
+            treasury: 'ChecksumEvmAddress',
+            incentives: 'ChecksumEvmAddress',
+    ) -> None:
+        super().__init__(
+            evm_inquirer=evm_inquirer,
+            base_tools=base_tools,
+            msg_aggregator=msg_aggregator,
+            pool_addresses=pool_addresses,
```

Comment:
does this initialize the classes correctly? both `Aavev3LikeCommonDecoder` and `SparkCommonDecoder` are called? My understanding was that there is some requirement about the classes calling super().__init__()